src/Functions/FunctionsHashingMisc.cpp

```diff
@@ -15,6 +15,8 @@ REGISTER_FUNCTION(Hashing)
     factory.registerFunction<FunctionSipHash64Keyed>();
     factory.registerFunction<FunctionSipHash128>();
     factory.registerFunction<FunctionSipHash128Keyed>();
+    factory.registerFunction<FunctionSipHash128Reference>();
+    factory.registerFunction<FunctionSipHash128ReferenceKeyed>();
```

Comment:
```suggestion
    factory.registerFunction<FunctionSipHash128Reference>({
        "Like [sipHash128](#hash_functions-siphash128) but implements the 128-bit algorithm from the original authors of SipHash.",
        Documentation::Examples{{"hash", "SELECT hex(sipHash128Reference('foo', '\\x01', 3))"}},
        Documentation::Categories{"Hash"}
    });
    factory.registerFunction<FunctionSipHash128ReferenceKeyed>({
        "Same as [sipHash128Reference](#hash_functions-siphash128reference) but additionally takes an explicit key argument instead of using a fixed key.",
        Documentation::Examples{{"hash", "SELECT hex(sipHash128ReferenceKeyed((506097522914230528, 1084818905618843912),'foo', '\\x01', 3));"}},
        Documentation::Categories{"Hash"}
    });
```